spec/requests/memory_spec.rb

```diff
@@ -1,7 +1,6 @@
 require 'rails_helper'
 
-describe "Memory Leak", :type => :request do
-
+describe "Memory Leak", :type => :request, :if => RUBY_ENGINE == 'ruby' do
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.